package.json

```diff
@@ -36,7 +36,7 @@
         "@babel/polyfill": "^7.10.4",
         "@babel/preset-env": "^7.16.11",
         "@fingerprintjs/fingerprintjs": "^3.3.3",
-        "asana": "^0.18.6",
+        "asana": "github:Asana/node-asana",
```

Comment:
Does this pull latest main branch? Could we pin this to a commit or tag?

Reply:
- This should pull the latest tags on GH.
- We had Asana issues which turned out to being unrelated to this but it is kinda weird they're not releasing to npm.
- Switching to this should have dependabot update for us as before and we pin in the package-lock file.
  - If there's concerns I think you can append @versionstring to the end here also.

Eitherway let me file an issue for fixing a deprecation with asana :)